### PR TITLE
Adds fields for RescanFolder and RefreshArtist

### PR DESCRIPTION
**Pull request by aetaric**
Just adding a few fields that are missing that I needed for checkrr.

### CODE REVIEW

**Comment by davidnewhall** [comment]
If you wanted more data about the commands being run, you could copy/paste `GetCommandStatus` methods into here as well. Example:
https://github.com/golift/starr/blob/master/sonarr/command.go#L90-L109

**Review by davidnewhall — APPROVED** [review]

**Comment by aetaric** [comment]
> If you wanted more data about the commands being run, you could copy/paste `GetCommandStatus` methods into here as well. Example: https://github.com/golift/starr/blob/master/sonarr/command.go#L90-L109

Mostly going for a fire and forget mindset here. Though that's not a terrible thing to add while I'm at it.

**Comment by aetaric** [comment]
This angers the CI.

**Comment by davidnewhall** [comment]
Run `make test` locally.

**Comment by davidnewhall** [comment]
The first 1 is always the most difficult. After that these PRs will come natural :D

File: lidarr/command.go [hunk 1]
```diff
@@ -82,3 +85,24 @@ func (l *Lidarr) SendCommandContext(ctx context.Context, cmd *CommandRequest) (*
 
 	return &output, nil
 }
+
+// GetCommandStatus returns the status of an already started command.
+func (s *Lidarr) GetCommandStatus(commandID int64) (*CommandResponse, error) {
```
Comment (davidnewhall) [inline lidarr/command.go:90]: the `s` needs to be `l`.

Comment (aetaric) [reply lidarr/command.go]: Yeah, I pulled it down and ran make test ;)

File: lidarr/command.go [hunk 2]
```diff
@@ -87,20 +87,20 @@ func (l *Lidarr) SendCommandContext(ctx context.Context, cmd *CommandRequest) (*
...
 	var output CommandResponse
 
 	if commandID == 0 {
 		return &output, nil
 	}
 
 	req := starr.Request{URI: path.Join(bpCommand, fmt.Sprint(commandID))}
-	if err := s.GetInto(ctx, req, &output); err != nil {
+	if err := l.GetInto(ctx, req, &output); err != nil {
 		return nil, fmt.Errorf("api.Post(%s): %w", &req, err)
```
Comment (davidnewhall) [inline lidarr/command.go:104]: This should say `api.Get(%s)` ... now I'm going to look at the example I pointed you to....

Comment (davidnewhall) [reply lidarr/command.go:104]: Yup Sonarr is wrong too. You're welcome to fix it too.

Comment (aetaric) [reply lidarr/command.go:104]: I mean, good news is, it passes the tests. Bad news is, you found another problem.

Comment (davidnewhall) [reply lidarr/command.go:104]: 1 small fix, sonarr too if you want, and I'll merge this up. :)

**Review by davidnewhall — APPROVED** [review]
Thank you!

**Pull request merged**